### PR TITLE
Adds giant spiders to spawner menu

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: Added giant spiders to spawner menu
/:cl:

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
I would like more optimisation because this will probably spam the spawn screen with a endless list in the case of spider-tide if they aren't split up & compressed properly.

- Brute (x4)
- Hunter (x3)
- Nurse (x7)

And others like a so.

**Comment by Cyberboss** [comment]
You're right @FantasticFwoosh, but that's a different PR

File: code/modules/mob/living/simple_animal/hostile/giant_spider.dm [hunk 1]
```diff
@@ -78,15 +80,17 @@
...
-		return 1
+		return TRUE
 	if(key)
 		to_chat(user, "<span class='notice'>Someone else already took this spider.</span>")
-		return 1
+		return TRUE
 	key = user.key
-	return 1
+	var/list/spawners = GLOB.mob_spawners[menu_name]
+	LAZYREMOVE(spawners, src)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/giant_spider.dm]: the GLOB. needs to be inside the LAZYREMOVE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — DISMISSED** [review]

**Comment by ShizCalev** [comment]
@MoreRobustThanYou compile.

**Comment by Jordie0608** [comment]
`code/modules/mob/living/simple_animal/hostile/giant_spider.dm:92:error: spawner.len: undefined var`
Stop using the web editor and maybe you could test the code works before you commit.

**Pull request closed without merging**